lib/network/modules/InteractionHandler.js

```diff
@@ -521,11 +513,11 @@ class InteractionHandler {
 
         // calculate the new scale
         let scale = this.body.view.scale;
-        let zoom = delta * (this.options.zoomSpeed / 10);
-        if (delta < 0) {
-          zoom = zoom / (1 - zoom);
-        }
-        scale *= (1 + zoom);
+        let direction = delta < 0 ? 1 : -1;
```

Comment:
Again if you don't change it don't use let, use const instead.

PS: I know that's the way it was but the old code is a mess. Don't take inspiration from that.